demo/vite.config.ts

```diff
@@ -18,7 +18,7 @@ export default defineConfig({
       apply: 'serve',
       configureServer: server => {
         server.middlewares.use((_req, res, next) => {
-          res.setHeader('Cross-Origin-Embedder-Policy', 'require-corp')
+          res.setHeader('Cross-Origin-Embedder-Policy', 'credentialless')
```

Comment:
Hum not sure what are the implications of that, help doesn't help but maybe I'm just tired ^^